File: workspace/judoku/src/main/java/judoku/data/Column.java
package judoku.data;

public class Column extends AbstractCellSet {
	public Column(int id) {
		super(id);
	}

	protected String getName() {
		return "column";
	}

	public String toString() {
        StringBuffer buf = new StringBuffer();
    	for (Cell c: this.getCells()) {
    		buf.append(c);
    		buf.append("\n");
    	}

        return buf.toString();
   }
}

File: workspace/scudoku/src/main/scala/scudoku/data/Column.scala
package scudoku.data

/**
 * Java2Scala port from judoku.data.Column
 */

class Column extends CellSet("column") {
  override def toString = {
      val buf: StringBuffer = new StringBuffer();
      for (c <- getCells) {
        buf.append(c)
        buf.append("\n")
      }

      buf.toString
  }
}